content/docs/tech-insights/displaying-fact-data/index.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Displaying Fact Data
+publishedDate: '2024-08-12T14:33:22.0Z'
+description: Roadie Plugins to display fact data directly
+---
+
+## Introduction
+
+Tech Insights in Roadie provides the capabilities to automatically retrieve and store relevant data for your entities. With this data you can enhance the Tech Insights visibility within your Roadie instance and easily keep track, visualize and display important information for your entities. 
+
+## Components
+
+Roadie provides various different components to display fact data. These range from overall fact gathering table views to more fine-grained fact info displaying widgets. The use cases for the former larger components are to provide and overview of retrieved fact data within the system, allowing users to keep themselves up to date with the latest information at a glance. The more fine-grained, entity level, fact displaying widgets can be used to display relevant current and historical information about collected facts. 
+
+
+
+### Tech Insights Fact Page
+
+Tech Insights Fact Page allows you to create a tabular view of facts collected to the system. These facts can range across different data sources. You can also define your own entity filtering to select only the relevant entities for this specific view.
```

Comment:
Can you put this "Tech Insights Fact Page" in some quotes or some highlighting to indicate its a name. Also might make sense to put "The" at the start of the sentence.